Description: enforce bash due to shopt usage
 shopt is a bash built-in.
Bug-Debian: https://bugs.debian.org/836401
Author: Fathi Boudra <fabo@debian.org>

---
 classes/ssl/ss_vncviewer |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/classes/ssl/ss_vncviewer
+++ b/classes/ssl/ss_vncviewer
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # ss_vncviewer:  wrapper for vncviewer to use an stunnel SSL tunnel
 #                or an SSH tunnel.
